babel.config.cjs

```diff
@@ -0,0 +1,27 @@
+const { declare } = require('@babel/helper-plugin-utils')
+
+module.exports = {
+  env: {
+    production: {
+      presets: ['@ownclouders/babel-preset']
+    },
+
+    test: {
+      presets: [
+        (module.exports = declare((api, options) => {
+          return {
+            presets: [
+              [
+                require('@ownclouders/babel-preset'),
+                {
+                  modules: 'cjs'
+                }
+              ]
+            ]
+          }
+        }))
+      ],
+      plugins: ['@babel/plugin-transform-modules-commonjs']
+    }
+  }
+}
```

Comment:
Could also use the preset from the monorepo

Reply:
It's using the preset but I needed to extend it to get tests working because the package is a module but tests are running cjs...

Reply:
Yeah, ok - let's not mess with that for now